Data ouput using single lumped K test

HaydenElza opened the pull request:
The data are now read in by rdgal and coerced into ppp objects using maptools.

HaydenElza committed:
Saving progress in case rain kill my computer

HaydenElza committed:
Pushing progress to work on another machine

HaydenElza committed:
Saving progress

HaydenElza committed:
Added maptools lib to handle as() coercion functions, fixed variable naming convention to _ instead of CamelCase, fixed general formatting, added as.ppp() for each dataset as is now working properly due to the inclusion of maptools

skochaver commented:
Both point clouds need to be in the same ppp object with the field x$marks identifying the type.

Check the documentation on the "Cross-type" Multitype K Function for clarification.
http://cran.r-project.org/web/packages/spatstat/spatstat.pdf

HaydenElza committed:
Fixed indentation, pared marks down to only what we need, combined the to point patterns into one

HaydenElza committed:
Fixed breaks in cut function, got Kest function working correctly, output table from Kest to csv, output plot of Kest to png

HaydenElza commented:
I used superimpose() to get both point clouds into the same ppp object. The marks are two leveled and are uniquely identifiable. The Kest() functions seems to be working, check that my parameters make sense. The output of Kest is exported to csv and the plot of the output is exported to png in the workspace.

## 

Here is a plot of the output:
![kest_plot](https://cloud.githubusercontent.com/assets/10215346/7303150/1bceb018-e9b4-11e4-8fed-654e84cc3fe2.png)

HaydenElza committed:
Added for loop through unique GRID_CODEs oupting tables and plots for each subset, changed the K test from Kest to Kcross as it more aptly fits our purpose

HaydenElza committed:
Added combine_csv.py to add together the separate csv files outside of R.

HaydenElza commented:
Fixed authors in commit history, feel free to pull now. I also added the python script I used to combine the csv files. The Ripley's K R script now uses Kcross() instead of Kest().

## 

Here is an example of the new output:
![k_subset_plot 53](https://cloud.githubusercontent.com/assets/10215346/7397729/01e7c138-ee70-11e4-9729-1ecc159ecc71.png)
